app/_layouts/collection.njk

```diff
@@ -38,7 +38,7 @@
         {% if pagination %}
           <section class="govuk-grid-column-two-thirds">
             <h2 class="govuk-heading-l govuk-!-font-size-27">
-              {{ app.posts.title or "Posts" }}
+              {{ posts.title or app.posts.title or "Posts" }}
```

Comment:
What's this doing?

Reply:
This allows you to overwrite the Posts title on the collection page in a page’s frontmatter, rather than setting it globally in app.json. I wanted to do it in the frontmatter than in app.json… but maybe it’s fine to do it there, and users can change/delete it? Not sure what’s the most obvious.